core/lib/core_web.ex

```diff
@@ -42,9 +43,8 @@ defmodule CoreWeb do
 
       import Phoenix.LiveView.Controller
 
-      plug(Systems.Project.BranchPlug)
+      # plug(Systems.Project.BranchPlug)
```

Comment:
Willen we commented out code toestaan in de repo?

Reply:
Die moet weer aangezet worden.

Reply:
Done